docs/development/l10n.rst

```diff
@@ -4,7 +4,8 @@ Getting Started with Weblate Translations
 SecureDrop is a whistleblower submission system for media
 organizations to securely accept documents from and
 communicate with anonymous sources. For more information, you may be
-interested to learn about :doc:`what makes SecureDrop unique
+interested to watch `The Global and Mail guide to using SecureDrop <https://www.youtube.com/watch?v=oSW2wMWtAMM>`_
```

Comment:
Nit: `Global and Mail` -> `Globe and Mail`

Reply:
:-) Interesting typo on my part. Good catch!